### PR TITLE
Remove and clarify jargon, fixes #10

### LINKED ISSUE #10: Double check for jargon
In the interest of attracting varied audiences to this article, I want us to be sure we double (and even triple) check for excessive jargon (related to both programming and education). One term that comes to mind is "forking."

### PR DESCRIPTION
**Pull request by gdevenyi**

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -275,10 +275,10 @@ \section*{Introduction}
 but besides that lecturers develop and improve their course materials in isolation.
 It is staggering to think how many wheels are being re-invented,
 how much valuable time has been wasted,
-and ultimately how the progress of tertiary education has been held back by
+and ultimately how the progress of postsecondary education has been held back by
```
Comment (gvwilson) [inline paper.tex]: post-secondary (?)

File: paper.tex [hunk 2]
```diff
@@ -275,10 +275,10 @@ \section*{Introduction}
...
 the lack of community collaboration and sharing in this process.
 This problem obviously extends beyond the university sector,
-but it is curious that it is endemic in tertiary education,
+but it is curious that it is endemic in postsecondary education,
```
Comment (gvwilson) [inline paper.tex]: post-secondary (?)

**Review by gvwilson — COMMENTED** [review]
Another hyphenation question, otherwise good to merge from me.

File: paper.tex (hunk 1 above)
Comment (gdevenyi) [reply paper.tex]: Yeah, should be that, will fix

File: paper.tex (hunk 2 above)
Comment (gdevenyi) [reply paper.tex]: Yeah, should be that, will fix

**Comment by gdevenyi** [comment]
PR updated.

**Pull request merged**